redhat-appstudio/infra-deployments
# Restructure staging builds installation directory

This PR restructures the build component

- moves all pipelines into build-templates
- remove experimental pipelines
- rename pipeline to docker-build
- move scripts to hack/build.sh
- document the app studio usage of default pipelines

## How to test
execute `hack/build/test-known-build.sh `which will run a build on a repo known to work.

This test will install pipelines and run a users namespace. 
See the readme in ./components/build for documentation
